src/screens/OtherProfile/index.js

```diff
@@ -439,7 +442,8 @@ const OtherProfile = () => {
     };
     const result = await setFollow(data);
     if (result.code === 200) {
-      fetchOtherProfile();
+      handleUpdateDiscoveryUser(other_id, true);
+      fetchOtherProfile(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, `fetchOtherProfile` is now called with `true` as an argument when following a user. This seems logical as you would want to get the most recent data after such an action. The `handleUpdateDiscoveryUser` function is also being called with new parameters. Ensure these changes align with your application logic.
```diff
- fetchOtherProfile();
+ handleUpdateDiscoveryUser(other_id, true);
+ fetchOtherProfile(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->